patches/tModLoader/Terraria/Item_tML.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Terraria.ModLoader.IO;
+
+namespace Terraria
+{
+	public partial class Item : TagSerializable
```

Comment:
k, I did some Google-ing and convinced myself it's an ok idea. I'm not sure how I feel about the _tML naming scheme though, but maybe it's the best idea for now.